src/index.tsx

```diff
@@ -4,11 +4,38 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+export interface User {
```

Comment:
Usually, we start the name of the interface with the letter _I_, for instance, _IUser, IStore, IAvatar_